float32 tau_x
float32 tau_y
float32 tau_z
float32 tau_yaw